locations/spiders/save_fr.py

```
@@ -0,0 +1,22 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class SaveFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "save_fr"
+    item_attributes = {"brand": "Save", "brand_wikidata": "Q121289450"}
+    sitemap_urls = ["https://magasin.save.co/sitemap.xml"]
+    sitemap_rules = [(r"https://magasin\.save\.co/.+/save\-[^/]+(\d+)$", "parse_sd")]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)